### PR TITLE
[bitnami/mongodb] Add resources for backup cronjob

### PR DESCRIPTION
**Pull request by PhilKuer**
### Description of the change

This adds the possibility to define resources for the mongodb `backup.cronjob`. 

### Benefits

Users can provide custom resources block for backup cronjob.

### Applicable issues

- fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mongodb/Chart.yaml [hunk 1]
```diff
@@ -39,4 +39,4 @@ maintainers:
 name: mongodb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
-version: 16.0.3
+version: 16.0.4
```
Comment (dgomezleon) [inline bitnami/mongodb/Chart.yaml:42]:
```suggestion
version: 16.1.0
```

**Review by dgomezleon — COMMENTED** [review]
The changes LGTM. Please bump a minor version since new values were added.

**Comment by PhilKuer** [comment]
Thanks. I have bumped to new minor version

**Review by dgomezleon — APPROVED** [review]
LGTM. Thanks for contributing!

**Pull request merged**